posts/2021-03-13-microprofile-config-2.0.adoc

```diff
@@ -31,7 +31,7 @@ Then, it's just 2 steps to launch:
 
 1. You’ll need to tell your build where all the awesome new features are defined. 
 +
-The easiest way to do this it to point to the link:https://search.maven.org/artifact/org.eclipse.microprofile/microprofile/4.0.1/pom[MicroProfile 4.0 Repository] which includes the link:https://search.maven.org/artifact/org.eclipse.microprofile.config/microprofile-config-api/2.0/jar[MicroProfile Config 2.0 API].
+The easiest way to do this it to point to the link:https://search.maven.org/artifact/org.eclipse.microprofile/microprofile/4.0.1/pom[MicroProfile 4.0 API] artifact which includes the link:https://search.maven.org/artifact/org.eclipse.microprofile.config/microprofile-config-api/2.0/jar[MicroProfile Config 2.0 API] artifact.
```

Comment:
Change,
"The easiest way to do this it to point to the link:https://search.maven.org/artifact/org.eclipse.microprofile/microprofile/4.0.1/pom[MicroProfile 4.0 API] artifact which includes the link:https://search.maven.org/artifact/org.eclipse.microprofile.config/microprofile-config-api/2.0/jar[MicroProfile Config 2.0 API] artifact."
to,
"The easiest way to do this it to point to the link:https://search.maven.org/artifact/org.eclipse.microprofile/microprofile/4.0.1/pom[MicroProfile 4.0 API] artifact that includes the link:https://search.maven.org/artifact/org.eclipse.microprofile.config/microprofile-config-api/2.0/jar[MicroProfile Config 2.0 API] artifact."

Reply:
This one stumped me for a while... But I think you're correct

Reply:
`The easiest way to do this it to point...` ->
`The easiest way to do this is to point...`